sirepo/package_data/static/js/srw.js

```diff
@@ -2533,6 +2534,20 @@ SIREPO.app.directive('simulationStatusPanel', function(appState, beamlineService
                 return $scope.model == 'coherentModesAnimation';
             }
 
+            function checkRunMode(runMode) {
+                return (runMode == 'sequential') || (runMode == 'parallel');
+            }
+
+            function coherentModesEnoughCores() {
+                var a = appState.models.coherentModesAnimation;
+                if (checkRunMode(a.jobRunMode)) {
```

Comment:
The failure can occur in parallel, too. I don't think it should ever be run sequentially.

Reply:
Yes. I believe you should get the number of cores correctly in dev.

Reply:
So '2 cores (SMP)'  should be a failure case?

Reply:
Yes.